specification/dsdl/grammar.tex

```diff
@@ -253,3 +253,15 @@ \subsection{Reserved identifiers}\label{sec:dsdl_reserved_identifiers}
     \texttt{lpt\textbackslash{}d}                      & \texttt{lpt9}      & Compatibility with Microsoft Windows \\
     \texttt{\_.*\_}                                    & \texttt{\_offset\_}& Special-purpose intrinsic entities \\
 \end{CyphalSimpleTable}
+
+\subsection{Reserved comment forms}\label{sec:dsdl_reserved_comment}
+
+Line comments that match the following regular expression are reserved for vendor-specific language extensions:
+\verb|^#\[.+\].*|
+
+\begin{remark}
+    \begin{minted}{python}
+        #[canadensis(enum_start)] all of this line is part of the reserved form.
```

Comment:
Minor note: Someone suggested `#[canadensis(enum_start)]`, but the directive I actually implemented is `#[canadensis(enum)]`.
To avoid confusion, I think it would be better to use `#[canadensis(enum)]`, or make up a different name for the example.

Reply:
updated